examples/get_sensor_history.py

```diff
@@ -155,8 +161,8 @@ def main():
         help="provide extremely verbose output.")
     parser.add_argument(
         '-i', '--include-value-time',
-        type=bool,
-        default=False,
+        dest="include_value_time", action="store_true",
+        default=True,
```

Comment:
Did you mean to change of default from False to True here?

Reply:
I was testing the output and left it `True` it is now set to `False`